test/suites/drift/suite_test.go

```diff
@@ -724,21 +677,4 @@ var _ = Describe("Drift", func() {
 			env.ConsistentlyExpectNoDisruptions(int(numPods), time.Minute)
 		})
 	})
-
-	It("should disrupt nodes that have drifted due to VNETSubnetID", func() {
```

Comment:
This should still be supportable, no? It just needs to be a valid subnet that replaces it for Drift?

We have at least some of the clients in tests now.

Reply:
this is replaced